mlrun/serving/remote.py

```diff
@@ -0,0 +1,156 @@
+import json
+
+import requests
+import storey
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
```

Comment:
Please change to `from urllib3.util.retry import Retry` there is not reason to import this from requests (there was at some point, now there's no longer a reason)
Also, we're about to do a refactor that will change all of our imports to `import X` instead of `from X import Y`
So you can replace also the import above already